errors.yaml

```diff
@@ -48,12 +52,24 @@ errors:
     description: "The Content-Type header does not match the request body payload format or the format is invalid."
   - code: missing_payload
     description: "The Content-Type header was specified, but no request body was sent to the server or the request body is empty."  
-  - code: dump_already_in_progress
-    description: "A dump creation is already in progress and a new one can't be triggered until the previous dump creation is not finished."
+  - code: dump_already_processing
+    description: "Dump creation is already in progress. A new dump creation process can be triggered when the current one is complete."
+  - code: dump_not_found
+    description: "The requested dump could not be found."  
   - code: dump_process_failed
     description: "An error occurred during dump creation process, task aborted."
   - code: invalid_ranking_rule
-    description: "The provided ranking rules are invalid. For more information on ranking rule configuration, check our [relevancy guide](https://docs.meilisearch.com/learn/core_concepts/relevancy.html#ranking-rules)"
+    description: "The provided ranking rules are invalid. This may be due to syntax errors in your ranking rules or specifying custom ranking rules on reserved fields or reserved expressions. For more information on ranking rule configuration, check our [relevancy guide](https://docs.meilisearch.com/learn/core_concepts/relevancy.html#ranking-rules)."
   - code: invalid_geo_field
     description: "The provided `_geo` field of one or more documents is invalid. Read more about `_geo` and how to troubleshoot it in [our dedicated guide](https://docs.meilisearch.com/reference/features/geosearch.html)."
+  - code: invalid_api_key
+    description: "The requested resources are protected with an API key. The provided API key is invalid. Read more about it at in our [dedicated guide](https://docs.meilisearch.com/reference/features/authentication.html)."
+  - code: invalid_store_file
+    description: "The `data.ms` folder is in an invalid state. Your `.mdb` file is corrupted or the `data.ms` folder has been replaced by a file."
+  - code: no_space_left_on_device
+    description: "The host system partition has reached its maximum capacity and can no longer accept writes." 
+  - code: database_size_limit_reached
+    description: "The requested database has reached its maximum limit; no further documents can be added. For information on customizing the maximum database size accepted by MeiliSearch, check out this [guide](https://docs.meilisearch.com/reference/features/configuration.html#max-udb-size)."
+  - code: index_primary_key_already_exists
+    description: "The requested index already has a primary key that cannot be changed."
```

Comment:
Are there primary keys that can be changed?

Reply:
Not that I know of.
```suggestion
    description: "The requested index already has a primary key and cannot be changed."
```

Reply:
Currently, if an index already has a primary key, it cannot be updated. 

In v0.24.0, this primary key can be updated if the index contains no document. In the rare case where there are 0 documents, this will reduce the current friction which is to delete the index and recreate it to change the primary key.

This is a change that was made at the end of last week very easily but which was not foreseen in the specification. However, as it can improve the user's life in a specific case, we decided to keep this change.

We didn't want to redefine a more precise error message for the moment on the core side but it will be done later. For the moment the message here is consistent because this error will occur only in case the primary key can't be changed.

Depending on what we decide to do about inference, waiting for a specific field like `id`, or being able to change the value of the primary key, it can change.